lib/SILOptimizer/Differentiation/PullbackCloner.cpp

```diff
@@ -1616,6 +1616,74 @@ class PullbackCloner::Implementation final
     builder.emitZeroIntoBuffer(uccai->getLoc(), adjDest, IsInitialization);
   }
 
+  void visitInjectEnumAddrInst(InjectEnumAddrInst *inject) {
+    SILBasicBlock *bb = inject->getParent();
+    SILValue origEnum = inject->getOperand();
+    bool unsupported = false;
+
+    // Only `Optional`-typed operands are supported for now. Diagnose all other
+    // enum operand types.
+    //
```

Comment:
Do we need those empty comment lines? Here and below?

Reply:
No, not really necessary. Removed them.